docker-compose.yml

```diff
@@ -3,12 +3,13 @@
 version: "3.9"
 services:
   al-nginx:
-    build:
-      context: al-nginx
-    image: al-nginx
+    image: nginx:latest
+    container_name: al-nginx
+    volumes:
+      - ./al-nginx/nginx.conf:/etc/nginx/nginx.conf
```

Comment:
The configuration is now added as a volume. Originally it was copied into the Docker image so change of config required rebuilding the image.